go/dhcp/config.go

```diff
@@ -340,6 +355,15 @@ func (d *Interfaces) readConfig() {
 						options[dhcp.OptionDomainNameServer] = ShuffleDNS(ConfNet)
 						options[dhcp.OptionRouter] = ShuffleGateway(ConfNet)
 						options[dhcp.OptionDomainName] = []byte(ConfNet.DomainName)
+						if portal.SecureRedirect == "enabled" {
+							if ConfNet.PortalFQDN != "" {
+								portalURL := "https://" + ConfNet.PortalFQDN + "/api"
+								options[dhcp.OptionCaptivePortal] = []byte(portalURL)
+							} else {
+								portalURL := "https://" + general.Hostname + "." + general.Domain + "/api"
```

Comment:
can we use something more meaningful than `/api` ?
Or is this path part of the spec.

Using `/rfc7710` would make it more explicit when analyzing logs